lib/ASTGen/Sources/ASTGen/BuiltinPound.swift

```diff
@@ -194,11 +194,98 @@ extension ASTGenVisitor {
     )
   }
 
-  func generateObjCSelectorExpr(freestandingMacroExpansion node: some FreestandingMacroExpansionSyntax) -> BridgedObjCSelectorExpr {
-    fatalError("unimplemented (objc selector)")
+  func generateObjCSelectorExpr(freestandingMacroExpansion node: some FreestandingMacroExpansionSyntax) -> BridgedExpr {
```

Comment:
Do we also need to diagnose the presence of trailing closure + generic arguments?